WordPress/src/main/java/org/wordpress/android/ui/posts/PostListItemViewHolder.kt

```diff
@@ -99,6 +112,16 @@ sealed class PostListItemViewHolder(
                 }
             }
         }
+
+        private fun isSafeClick(view: View): Boolean {
```

Comment:
Can we just add a comment in code explaining why we need to check for isSafeClick? (i.e. to prevent 2 Editors from being launched simultaneously and then producing a crash, etc)

Reply:
I have added the comment!